sql/moz-fx-data-shared-prod/acoustic_external/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: Acoustic External
+description: |-
+  This dataset holds all data coming from Acoustic. Either via Fivetran or one time data dumps of tables from Acoustic.
+  As part of the migration to Braze, all the data from Acoustic will be exported and uploaded to BigQuery once.
+dataset_base_acl: restricted
+user_facing: false
+labels: {}
+workgroup_access:
```

Comment:
Is it because it's restricted that you don't add ` - workgroup:mozilla-confidential` as viewer?
This is the workgroup that makes it accessible, also from VMs, for managed backfills.